btk/obs_conditions.py

```diff
@@ -3,18 +3,76 @@
 import numpy as np
 import galsim
 import descwl
+from collections import namedtuple
 
+# A simple class archetype to serve as a dictionary without having to write the field names every time
+Survey = namedtuple("Survey", ["name", "pixel_scale", "psf_scale", "bands", "mean_sky_level", "exp_time", "zero_point"])
 
-all_surveys = {
-    "LSST": {
-        "name": "LSST",
-        "bands": ("y", "z", "i", "r", "g", "u"),
-        "pixel_scale": 0.2,
-    },
-    "DES": {"name": "DES", "bands": ("i", "r", "g", "z"), "pixel_scale": 0.263},
-    "CFHT": {"name": "CFHT", "bands": ("i", "r"), "pixel_scale": 0.185},
-    "HSC": {"name": "HSC", "bands": ("y", "z", "i", "r", "g"), "pixel_scale": 0.17},
-}
+pix_ROMAN = 0.11
+pix_RUBIN = 0.2
+pix_HST = 0.06
+pix_EUCLID = 0.101
+pix_HSC = 0.167
+pix_DES = 0.263
+pix_CFHT = 0.185
+
+# Sigma of the psf profile in arcseconds.
+sigma_ROMAN = 0.11 * np.array([1.68, 1.69, 1.86, 2.12, 2.44, 2.71])  # https://arxiv.org/pdf/1702.01747.pdf Z-band
+sigma_RUBIN = np.array([0.327, 0.31, 0.297, 0.285, 0.276, 0.267])  # https://www.lsst.org/about/camera/features
+sigma_EUCLID = np.array([0.16])  # https://sci.esa.int/documents/33859/36320/1567253682555-Euclid_presentation_Paris_1Dec2009.pdf
+sigma_HST = np.array([0.074])  # Source https://hst-docs.stsci.edu/display/WFC3IHB/6.6+UVIS+Optical+Performance#id-6.6UVISOpticalPerformance-6.6.1 800nm
+sigma_HSC = np.array([0.306, 0.285, 0.238, 0.268, 0.272])  # https://hsc-release.mtk.nao.ac.jp/doc/ deep+udeep
+
+
+Euclid = Survey("Euclid",
+                pix_EUCLID,
+                sigma_EUCLID,
+                ["VIS"],
+                np.array([22.9]),
+                np.array([2260]),
+                np.array([6.85]))
```

Comment:
I notice the Euclid survey is currently only supporting the visible band ; we should add support for NIR bands in the future (as they have a lower resolution, I think the best way to do it would be to have two separate surveys Euclid_NIR and Euclid_VIS).

Reply:
Eeeewww tricky because then we run into the issue of having different resolutions for different bands. I'd punt on that or create a different survey object.

Reply:
We can discuss this further after this PR is merged, but I think two different surveys will work well? Not sure why different resolutions in different bands is an issue (since we have WCS info)

Reply:
Yep, attach the pixel scale to the band rather than the Survey and you should be fine.

Reply:
OR add an Instrument layer in between but that makes it heavier

Reply:
What would be the result if we had different resolutions within the same survey ? Some kind of weirdly shaped numpy array ? Currently for one survey we get a stamp_size*stamp_size*number_of_bands numpy array for each blend (+isolated_galaxies...), and when we have different surveys we get several multiband arrays (one for each survey).

Reply:
Yeah so two survey objects is the easy way to go about. I like the idea of having an instrument layer in the survey object, but then we need to be careful about multi-resolution and tracking astrometry (so the wcs needs to be matched well) and then it poses implementation issues for the deblenders that deal, or not with this. This is actually a functionality that we were thinking of adding so it's coming.

Reply:
Actually do we need a `Survey`? For image simulation and blending purposes, only the Instrument is relevant. But it could incorporate the survey in its name e.g. `Euclid_VIS` and `Euclid_NIR`

Reply:
I am not sure what you mean but things like exposure time belong to a survey more than an instrument.

Reply:
On the contrary, exposure time is more a feature of the bandpass (filter) than instrument. This is the reason why the images in the various bands will not end up with the same depth.

Reply:
I guess we can think about that further once this PR gets merged.

Reply:
I'm with Thomas here, the exposure time depends on the survey design. Even though it might be different for each band. Anyway. I'll close this conversation and open an issue.